x/wasmplus/handler.go

```diff
@@ -29,8 +29,6 @@ func NewHandler(k wasmtypes.ContractOpsKeeper) sdk.Handler {
 				)
 				res, err = msgServer.StoreCodeAndInstantiateContract(sdk.WrapSDKContext(ctx), msg2)
 				return sdk.WrapServiceResult(ctx, res, err)
-			} else {
-				return nil, err
```

Comment:
Could you please tell me why this line was removed in this commit?

Reply:
reflect golangci-lint's direction

Reply:
golangci-lint error is like this?
```
x/wasmplus/handler.go:31:11: indent-error-flow: if block ends with a return statement, so drop this else and outdent its block (revive)
                        } else {
                                return nil, err
                        }
```

so I think NOT delete L33.


FYI:https://github.com/Finschia/wasmd/pull/36/commits/f3f0bb9161ce454a5188f8d4f0bae9aa52a15ecc

Reply:
If we delete it, returning in L34 (https://github.com/Finschia/wasmd/pull/35/commits/d0538260771bed6dab0a47973e5aa17b37fe0b74#diff-91d00bcd1785e97f20e683fb5778f603895fefacb93ef9952a311d76705814faR34) happens instead of this. This return as the same value we deleted.

Reply:
@da1suk8 As you said, my fix is not according to the point of

> indent-error-flow


So, I will fix https://github.com/Finschia/wasmd/issues/32 in this PR and make it simple how to branch the logic.